Multisig/UI/Safe Management/Create Safe/GoogleWeb3AuthLoginModel.swift

```diff
@@ -3,42 +3,46 @@ import CustomAuth
 import UIKit
 
 class GoogleWeb3AuthLoginModel {
-    var onClose: ((_ key: String?, _ email: String?) -> Void)
+    var authorizationComplete: () -> Void
+    var keyGenerationComplete: ((_ key: String, _ email: String?) -> Void)
```

Comment:
no need to define parameter names ((String, String?) -> Void) should be enough

Reply:
I find this more readable. And easier to understand. Is there a disadvantage to it?